google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudConfigurableTest.java

```diff
@@ -43,7 +43,7 @@
   private TextFieldWithBrowseButton cloudSdkDirectoryField;
 
   private static final String CLOUD_SDK_DIR_PATH = new File("/a/b/c/gcloud-sdk").getAbsolutePath();
-  private static final String MISSING_SDK_DIR_WARNING = "Cloud SDK home directory is not correct.";
+  private static final String INVALID_SDK_DIR_WARNING = "Cloud SDK home directory is not correct.";
```

Comment:
I still think you want to be more explicit about what's 'not correct' about the Cloud SDK home directory.